src/Rental.sol

```diff
@@ -287,11 +282,11 @@ contract Rental {
         uint256 withdrawableCollateral;
         uint256 timeLeftUntilFullyPaid = rentalStartTime + collateralPayoutPeriod - block.timestamp;
         if (timeLeftUntilFullyPaid > 0) {
-            withdrawableCollateral = address(this).balance.sub(collateral.mul(timeLeftUntilFullyPaid.div(collateralPayoutPeriod)));
+            withdrawableCollateral = address(this).balance - (collateral * timeLeftUntilFullyPaid) / collateralPayoutPeriod;
```

Comment:
I think the parentheses around collateral and timeLeftUntilFullyPaid can be removed. 

withdrawableCollateral = address(this).balance - collateral * timeLeftUntilFullyPaid / collateralPayoutPeriod;

Reply:
ya totally, can do - just left for verbosity